libcontainer/rootfs_linux.go

```diff
@@ -326,6 +330,53 @@ func mountCgroupV2(m *configs.Mount, c *mountConfig) error {
 	})
 }
 
+var (
+	syscallOnce sync.Once
+	syscallErr  error
+)
+
+func mountByMove(m *configs.Mount, dest string) error {
+	// Check once for all mount point that syscall exists
+	syscallOnce.Do(func() {
```

Comment:
Could this be exported as an interface so other parts may query the syscall availability status? We face the same issue too and seems it could be reused.